frontend/lib/utils/task.dart

```diff
@@ -1,14 +1,16 @@
 import 'package:frontend/utils/services.dart';
 
 class Task {
-  final String? title;
-  final ActionInfo? action;
-  final String? author;
-  final List<ReactionInfo> reactions;
-  final int numberOfUsers;
-  final bool isActive;
+  String? workflowId;
+  String? title;
+  ActionInfo? action;
+  String? author;
```

Comment:
Why always optional?

Reply:
Its is mandatory for the app to proper work (mainly usefull for creating new task)